li-apache-kafka-clients/build.gradle

```diff
@@ -15,6 +15,7 @@ plugins {
 dependencies {
     compile "com.linkedin.kafka:kafka-clients:${rootProject.ext.liKafkaVersion}"
     compile "com.linkedin.mario:mario-client-all:${rootProject.ext.marioVersion}"
+    compile "com.googlecode.concurrent-locks:concurrent-locks:1.0.0"
```

Comment:
This library has one contributor and has not changed in 5 years.  Is this really a good library to use?